soc/espressif/common/loader.c

```diff
@@ -264,7 +264,8 @@ void __start(void)
 	}
 #endif
 
-#if !defined(CONFIG_MCUBOOT) && !defined(CONFIG_SOC_ESP32S3_APPCPU)
+#if !defined(CONFIG_SOC_ESP32S3_APPCPU) && !defined(CONFIG_SOC_ESP32S3_APPCPU) && \
```

Comment:
does this condition repeat CONFIG_SOC_ESP32S3_APPCPU twice?

Reply:
thx, should be esp32 && esp32s3. Fixed